testutil/base.go

```diff
@@ -34,6 +36,13 @@ func (s *BaseTest) SetUpTest(c *check.C) {
 	if len(s.cleanupHandlers) != 0 {
 		panic("BaseTest cleanup handlers were not consumed before a new test start, missing BaseTest.TearDownTest call?")
 	}
+
+	// When unit tests are called with SNAPD_DEBUG=1 , we tend to get some failures due to the
```

Comment:
Thanks for this comment! One nitpick about the spacing: `output . ` should probably be `output.` (or is there a reason for the extra space?) (same in some other place with a ` , `).